js/src/components/google-combo-account-card/connect-ads/useCreateAccountActions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useUpsertAdsAccount from '.~/hooks/useUpsertAdsAccount';
+
+const useCreateAccountActions = () => {
+	const [ creatingNewAccount, setCreatingNewAccount ] = useState( false );
+	const [ showCreateNewModal, setShowCreateNewModal ] = useState( false );
```

Comment:
@ankitrox I would move the `showCreateNewModal` logic outside of the hook since the hook should only be responsible to create a new account. Having the logic to show/hide the modal should not be part of it.

Reply:
I think this hook is not at all required and we can move everything in this hook inside `connectAds` component.